tests/cases/compiler/classImplementsPrimitive.ts

```diff
@@ -0,0 +1,17 @@
+// classes cannot implement primitives
+
+class C implements number { }
+class C2 implements string { }
+class C3 implements boolean { }
+class C4 implements Void  { }
+class C4a implements void {}
+class C5 implements Null { }
+class C5a implements null { }
+class C6 implements undefined { }
+class C7 implements Undefined { }
+
+enum E { A }
+class C8 implements E { }
+
+class C4a implements void {}
+class C5a implements null { }
```

Comment:
same comment for these; it's nice to have these tests, but if they're already in the test suite, you don't need to add them.

Reply:
I tried to look up the case of class implementing enum. However there isn't any result found. this time, I would remove these cases because these tests aren't relevant to this PR